### DISCUSSION TITLE
Max Listener Error

### DISCUSSION THREAD

**Discussion by dodeyeikoi** [Cyfrin/foundry-full-course-cu#986] [Q&A]
I started using thirdweb deploy to try out some of my contracts but After deploying a few times, I now get this error:

````
(node:3233) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [TLSSocket]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
`````
how can I solve this please?

**Comment by alfheimrShiven** [comment]
Hey @dodeyeikoi are you still facing this issue? If no, requesting you to add the solution and close the thread. Thanks!

**Reply by dodeyeikoi** [reply]
I've been able to resolve it, I removed the ports I wasn't using anymore in my vscode editor and closed and opened it again, I also used ````npx thridweb create```` to install some packages that were not installed
